cloud-azure/src/test/java/com/sequenceiq/cloudbreak/cloud/azure/AzureUtilsTest.java

```diff
@@ -15,6 +15,7 @@
 import java.util.Map;
 
 import org.jetbrains.annotations.NotNull;
```

Comment:
I think the agreement in Slack was not to use these JetBrains annotations at all.

Reply:
Sorry, my bad, I will fix it.